20-packages.ks

```diff
@@ -1,4 +1,4 @@
-%packages --excludedocs
+%packages --excludedocs --inst-langs en_US.utf8
```

Comment:
TIL `--inst-langs` is a thing now, as confirmed by reading https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/automatically_installing_rhel/kickstart-script-file-format-reference_rhel-installer#common-package-selection-options_package-selection-in-kickstart